src/main/webapp/app/shared/user-settings/vcs-access-tokens-settings/vcs-access-tokens-settings.html

```diff
@@ -0,0 +1,130 @@
+                                [dialogError]="dialogError$"
+                            >
+                                <fa-icon [icon]="faTrash" />
+                                <div jhiTranslate="entity.action.delete" class="ms-2"></div>
+                            </button>
+                        </td>
+                    </tr>
+                </tbody>
+            </table>
+            } @else {
+            <div jhiTranslate="artemisApp.userSettings.vcsAccessTokensSettingsPage.noTokenSet"></div>
+            }
+        </dt>
+    </div>
+    @if (!this.currentUser?.vcsAccessToken) {
+    <div class="list-group-item">
+        <dt>
+            <div class="btn-group" role="group" aria-label="Actions">
+                <jhi-button
+                    class="d-flex"
+                    id="add-new-token-button"
+                    [btnType]="ButtonType.PRIMARY"
+                    [btnSize]="ButtonSize.SMALL"
+                    [icon]="faEdit"
+                    [title]="'artemisApp.userSettings.vcsAccessTokensSettingsPage.addToken'"
+                    (onClick)="addNewVcsAccessToken()"
+                />
+            </div>
+        </dt>
+    </div>
+    } } @if (edit) {
+    <div class="list-group-item">
+        <dt>
+            <h4>Add personal access token</h4>
+
+            <div class="row mb-3">
+                <div class="col-sm-6">
+                    <jhi-date-time-picker
+                        [shouldDisplayTimeZoneWarning]="false"
+                        labelName="{{ 'artemisApp.userSettings.vcsAccessTokensSettingsPage.expiryDate' | artemisTranslate }}"
+                        [(ngModel)]="expiryDate"
+                        (valueChange)="validateDate()"
+                        [error]="!!expiryDate && !validExpiryDate"
+                        name="expiryDate"
+                        id="expiryDate"
+                    />
+                </div>
+            </div>
+
+            <div class="btn-group" role="group" aria-label="Actions">
+                <jhi-button
+                    class="d-flex"
+                    id="create-vcs-token-button"
+                    [btnType]="ButtonType.PRIMARY"
+                    [btnSize]="ButtonSize.SMALL"
+                    [disabled]="!validExpiryDate"
+                    [icon]="faSave"
+                    [title]="'artemisApp.userSettings.vcsAccessTokensSettingsPage.createAccessToken'"
+                    (onClick)="sendTokenCreationRequest()"
+                    style="margin-right: 3px"
+                />
+                <jhi-button
+                    class="d-flex"
+                    id="cancel-vcs-token-creation-button"
+                    [btnType]="ButtonType.SECONDARY"
+                    [btnSize]="ButtonSize.SMALL"
+                    [icon]="faBan"
+                    [title]="'entity.action.cancel'"
+                    (onClick)="cancelTokenCreation()"
+                />
```

Comment:
Can you check if we really need to set `d-flex` as class on these buttons?
Also, for the first button, you may be able to get the margin you want by using the `me-1` bootstrap class

Reply:
nice tip, for these two buttons I could remove the `d-flex`, and adding the `me-1` worked :+1: